build_tools/packaging/python/generate_release_index.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""Creates an HTML page for releases for `pip install --find-links` from a subdirectory in an S3 bucket.
+
+Sample usage:
+
+    ```bash
+    ./build_tools/packaging/python/generate_release_index.py \
+        --bucket=therock-dev-python \
```

Comment:
Can you list known bucket names in the docstring here? The workflow uses `--bucket=therock-${{ env.RELEASE_TYPE }}-python`.